### PR TITLE
NET-1603: Manage DNS NM changes

### PR DESCRIPTION
**Pull request by yabinma**
## Checklist before requesting a review
- [ ] My changes affect only 10 files or less.
- [ ] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netmaker is awesome.

### CODE REVIEW

File: mq/mq.go
Comment (abhishek9686) [inline mq/mq.go:1]: remove periodic sync check, instead this can be published when peer update is pushed

File: mq/publishers.go [hunk 1]
```diff
@@ -249,3 +250,48 @@ func sendPeers() {
 		}
 	}
 }
+
+func sendDNSSync() error {
```
Comment (abhishek9686) [inline mq/publishers.go:254]: this func should be simplified to all get all entries from the DB and push it to clients

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: mq/mq.go
Comment (yabinma) [reply mq/mq.go:1]: Move to peerUpdate in new commit. please review again.

File: mq/publishers.go (hunk 1 above)
Comment (yabinma) [reply mq/publishers.go:257]: Removed the cache in new commit. Please help review again.

**Comment by abhishek9686** [comment]
add relevant PR title

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**